src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
+
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    Boolean update(MovieSession movieSession);
+
+    Boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` if the operation's success can be determined by exceptions or other means. Returning a `Boolean` might not be necessary.